www/Themes/Centreon-Dark/variables.css

```diff
@@ -29,7 +29,7 @@
     --table-form-header-h3-font-color: var(--body-color);
     --form-table-list-lvl-1-background-color: var(--centreon-popin-background-color);
     --form-table-list-lvl-1-col-name-background-color: var(--centreon-popin-background-color);
-    --form-table-list-lvl-1-col-name-border-bottom-color: var(--body-color);
+    --form-table-list-lvl-1-col-name-border-bottom-color: #776e62;
```

Comment:
`#776e62` seems used several times, could you use a variable for it ?

Reply:
@kduret done :D !